.github/workflows/publish-test-result-from-fork.yml

```diff
@@ -16,46 +16,9 @@ jobs:
          github.event.sender.login == 'dependabot[bot]' ||
          github.event.workflow_run.head_repository.full_name != github.repository
       )
-    steps:
-      - name: Download Artifacts
-        uses: actions/github-script@v3
-        with:
-          script: |
-            var fs = require('fs');
-            var path = require('path');
-            var artifacts_path = path.join('${{github.workspace}}', 'artifacts')
-            fs.mkdirSync(artifacts_path, { recursive: true })
-            var artifacts = await github.actions.listWorkflowRunArtifacts({
-               owner: context.repo.owner,
-               repo: context.repo.repo,
-               run_id: ${{ github.event.workflow_run.id }},
-            });
-            for (const artifact of artifacts.data.artifacts) {
-               var download = await github.actions.downloadArtifact({
-                  owner: context.repo.owner,
-                  repo: context.repo.repo,
-                  artifact_id: artifact.id,
-                  archive_format: 'zip',
-               });
-               var artifact_path = path.join(artifacts_path, `${artifact.name}.zip`)
-               fs.writeFileSync(artifact_path, Buffer.from(download.data));
-               console.log(`Downloaded ${artifact_path}`);
-            }
-      - name: Extract Artifacts
-        run: |
-          for file in artifacts/*.zip
-          do
-            if [ -f "$file" ]
-            then
-              dir="${file/%.zip/}"
-              mkdir -p "$dir"
-              unzip -d "$dir" "$file"
-            fi
-          done
-      - name: Publish Linux UT Results
-        uses: EnricoMi/publish-unit-test-result-action@v1
-        with:
-          commit: ${{ github.event.workflow_run.head_sha }}
-          files: artifacts/linux-ut-result-*/**/*.xml
-          check_name: Linux Test Report
-          comment_title: Linux Test Report
+    uses: ./.github/workflows/publish-test-result-from-fork-action.yml
```

Comment:
seem not work because it can't find the file
see https://github.com/aceforeverd/fedb/actions/runs/3224277329/jobs/5275245948

Note those changes are bit complex to test, because workflow from `workflow_dispatch` only apply changes after changes merged into main branch. You may test over your own repo first. Here is the steps:

1. merge changes of this PR into main (or any default branch you set) into your own repo
2. create a pull request into your repo that triggers `cicd` , `hybridse` and `sdk` workflows
3. see if the three test report will appear in PR comment

here is my PR you can checkout https://github.com/aceforeverd/fedb/pull/2